pom.xml

```diff
@@ -198,6 +197,12 @@
                 <artifactId>json</artifactId>
                 <version>20090211</version>
             </dependency>
```

Comment:
Ah, the way it was emailed (and looks in github), I thought it was to the previous lines.  Maybe comments are "supposed" to be after the code?

I just did a "maven clean install" on ahousing/gateway, followed by "maven clean verify" on enterprise.gateway.  It's fine.  The reason, I believe, is that [enterprise.gateway/pom.xml](https://github.com/kaazing/enterprise.gateway/blob/develop/pom.xml) first lists a dependency on gateway.bom, but the second dependency is on gateway.